driver/src/contracts/dfusion.rs

```diff
@@ -166,6 +166,19 @@ impl SnappContract for SnappContractImpl {
             .map_err(DriverError::from)
     }
 
+    fn calculate_order_hash(&self, slot: U256, standing_order_index: Vec<U128>) -> Result<H256> {
```

Comment:
This was moved because of a warning that the trait functions were not implemented in the order they were defined.